### PR TITLE
Update extended attributes

### PR DESCRIPTION
**Pull request by AndrewRTsao**
## What are you changing in this pull request and why?

Add an additional bullet to clarify that extended attributes only support top-level keys / fields. This will prevent confusion in case users try to pass in partial or individual sub-key values (instead of passing in the entire top-level key as a nested JSON block in the YAML). The latter will work while the former will cause an error when you actually try to run dbt.

### CODE REVIEW

**Comment by AndrewRTsao** [comment]
Updating our docs to clarify this point per a recent discussion with @ericbaumann 

FYI @Fleid

**Review by Fleid — APPROVED** [review]

File: website/snippets/_cloud-environments-info.md [hunk 1]
```diff
@@ -65,6 +65,8 @@ If you're developing in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in
 
 - If the attribute doesn't exist, it will add the attribute or value pair to the profile. 
 
+- Only **top-level** keys are accepted in extended attributes. In other words, if you wish to update a particular *sub-key value*, you will need to update the entire top-level key as a JSON block in your resulting YAML. For example, assuming you wish to pass in a custom override for a specific individual [service account JSON field](/docs/core/connect-data-platform/bigquery-setup#service-account-json) for your BigQuery connection (e.g. `project_id` or `client_email`), you will need to pass in an override for the entire top-level `keyfile_json` key / attribute instead via extended attributes (with the sub-fields passed in as a nested JSON block).
```
Comment (mirnawong1) [inline website/snippets/_cloud-environments-info.md]: @AndrewRTsao , tweaked this to simplify a little

```suggestion
Only the **top-level keys** are accepted in extended attributes. This means that if you want to change a specific sub-key value, you must provide the entire top-level key as a JSON block in your resulting YAML. For example, if you want to customize a particular field within a [service account JSON](/docs/core/connect-data-platform/bigquery-setup#service-account-json) for your BigQuery connection (like 'project_id' or 'client_email'), you need to provide an override for the entire top-level 'keyfile_json' main key/attribute using extended attributes. Include the sub-fields as a nested JSON block.
```

File: website/snippets/_cloud-environments-info.md [hunk 2]
```diff
@@ -65,6 +65,8 @@ If you're developing in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in
 
 - If the attribute doesn't exist, it will add the attribute or value pair to the profile. 
 
+Only the **top-level keys** are accepted in extended attributes. This means that if you want to change a specific sub-key value, you must provide the entire top-level key as a JSON block in your resulting YAML. For example, if you want to customize a particular field within a [service account JSON](/docs/core/connect-data-platform/bigquery-setup#service-account-json) for your BigQuery connection (like 'project_id' or 'client_email'), you need to provide an override for the entire top-level 'keyfile_json' main key/attribute using extended attributes. Include the sub-fields as a nested JSON block.
```
Comment (mirnawong1) [inline website/snippets/_cloud-environments-info.md]:
```suggestion
Only the **top-level keys** are accepted in extended attributes. This means that if you want to change a specific sub-key value, you must provide the entire top-level key as a JSON block in your resulting YAML. For example, if you want to customize a particular field within a [service account JSON](/docs/core/connect-data-platform/bigquery-setup#service-account-json) for your BigQuery connection (like 'project_id' or 'client_email'), you need to provide an override for the entire top-level `keyfile_json` main key/attribute using extended attributes. Include the sub-fields as a nested JSON block.
```

**Review by mirnawong1 — APPROVED** [review]
🚢 thanks for opening this up @AndrewRTsao ! merging this for you, looks great!

**Pull request merged**